### PR TITLE
Improve visual aesthetics

### LINKED ISSUE #18: Presentation: General visual aesthetics improvements
Improve the visual appeal of the website, including:
- [ ] Formatting
- [ ] Fixing the footer's text alignment (#2)
- [ ] Typefaces
- [ ] Miscellaneous

### PR DESCRIPTION
**Pull request by Coedice**
Closes: https://github.com/eduvis/eduvis.github.io/issues/18

### CODE REVIEW

File: _config.yml
Comment (Coedice) [inline _config.yml:26]: How do you feel about this tagline?

File: _config.yml
Comment (Coedice) [inline _config.yml:34]: Do we have any social media accounts we could add here?

File: assets/css/custom.scss
Comment (Coedice) [inline assets/css/custom.scss:27]: Since there isn't a lot of sass in this repo (since the bulk of it is from the `texture` template), I think we can just keep all template-override scss in this `custom.scss`

File: _config.yml
Comment (njsch) [reply _config.yml:26]: Yeah I think that should be fine.

File: _config.yml
Comment (njsch) [reply _config.yml:34]: We only have Github and email addresses. There is also a thus-far unused and somewhat primitive HTML contact file in `_includes`.

Comment (Coedice) [reply _config.yml:34]: Oh ok, I'll add those

File: assets/css/custom.scss
Comment (njsch) [reply assets/css/custom.scss:27]: Yeah, certainly if it's easier and more convenient.

**Review by njsch — DISMISSED** [review]

**Comment by njsch** [comment]
@Coedice, how come I cannot merge without explicitly manually overriding Branch protection rules?

**Comment by Coedice** [comment]
@njsch It could be because there are unresolved threads (which is right, because there are changes to be made). Typically it is the assignee who hits the merge button once the reviewer has approved the MR

**Comment by njsch** [comment]
Also, where is the scri[t that used to be in the header (not head) for the customisable Google search box?

**Comment by Coedice** [comment]
@njsch I forgot to call this out. I had removed it, since I figured we don't have all that many pages on the website anyway.

It's also very a very ugly search bar.

Do you think we should keep it? I could add it back and see if I can improve how it looks if you think it's worth having

**Comment by njsch** [comment]
Oh okay. In that case, we may as well just leave it out. Thanks for clarifying.On 23 Nov 2023, at 12:13 pm, Gavin Douch ***@***.***> wrote:﻿
@njsch I forgot to call this out. I had removed it, since I figured we don't have all that many pages on the website anyway.
It's also very a very ugly search bar.
Do you think we should keep it? I could add it back and see if I can improve how it looks if you think it's worth having

**Comment by njsch** [comment]
I have a hunch about this: is the 3rd-party Texture theme contained anywhere on Github or in Jekyll other than [here](https://github.com/samarsault/texture)?

Unless you're wanting it to come from somewhere else, you have to add it explicitly as a remote theme which does not appear to have been done. It might be good to compare `samarsault/texture/_config.yml` side-by-side with the Eduvis one as I think the switch from [Minima](https://github.com/jekyll/minima) to Texture has possibly corrupted it.

I'm sure it's not difficult to fix but this is why I have hesitated to change the theme from Minima on my personal site.

**Comment by njsch** [comment]
I've tried to fix it but it still won't deploy. Sorry if I should have made my own branch off yours, otherwise not sure if you may have to manually fix some merge conflicts on your own end. But the YAML file looks more consistent with what's in the main Texture repo now.

**Comment by njsch** [comment]
Still isn't working even after attempting to fix according to `samarsault/texture` issue [#43](https://github.com/samarsault/texture/issues/43).

**Comment by Coedice** [comment]
It looks like GitHub Pages only supports a small list of themes: https://pages.github.com/themes/

I will modify this PR to use [cayman](https://pages-themes.github.io/cayman/) instead

**Comment by njsch** [comment]
Unfortunately, because I had dismissed my previous review, it appears that I couldn't re-review. So I have done a manually overridden merge commit. I know it's not ideal, maybe I should've re-requested a review myself. But I wasn't exactly sure how else to do this

**Pull request merged**
